store.go

```diff
@@ -174,7 +174,11 @@ func (c *controller) watchNetworks() error {
 	cs := c.store
 	c.Unlock()
 
-	nwPairs, err := cs.KVStore().WatchTree(datastore.Key(datastore.NetworkKeyPrefix), nil)
+	networkKey := datastore.Key(datastore.NetworkKeyPrefix)
+	if err := ensureKeys(networkKey, cs); err != nil {
+		return err
```

Comment:
Instead of returning naked err value which is probably two low level can we replace that with
``` 
return fmt.Errorf("failed to ensure if the keys are valid and present in store: %v", err)
```

Makes debugging user problems much easier. Same for below.